templates/wireguard_conf.epp

```diff
@@ -46,10 +46,11 @@ MTU=<%= $mtu %>
 
 <% $peers.each |$peer| { -%>
 
+[Peer]
 <% if $peer['description'] { -%>
-# <%= $peer['description'] %>
+# friendly_name = <%= $peer['description'] %>
```

Comment:
I like the idea of building compatibility for the prometheus exporter. Is friendly_name documented somewhere?